assets/js/src/common/typography/list/list.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+type Props = {
+  children: React.ReactNode,
+  isOrdered?: boolean,
+};
+
+const List = ({
+  children,
+  isOrdered,
+}: Props) => {
+  const Element = isOrdered ? 'ol' : 'ul';
+  return React.createElement(
```

Comment:
I've seen this in a couple of new components so just a note - we should probably pick one style and use it. I'd vote for `<Element...>` since it's the syntax we normally use (and this is a `.tsx` file).

Reply:
Unified, also updated `<Heading>` component